MetaMask/metamask-extension
# Allow user to turn off ENS DNS resolution

## Explanation

Allows the user to toggle off IPFS domain resolution

* Fixes https://github.com/MetaMask/MetaMask-planning/issues/925

## ToDo
- [ ] Need content to finalize this PR

## Screenshots/Screencaps

Coming soon.

## Manual Testing Steps

1.  Toggle the setting on
2. Open a new tab and type any known ".eth" ENS name
3. See the URL change to `https://app.ens.domains/${whatever-ens-name}`
4. Toggle the setting off
5. Open a new tab and type any known ".eth" ENS name
6. Default 404 for browser behavior occurs

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
